introduction.md

```diff
@@ -1,3 +1,10 @@
 Hi I'm Nietzsche
 best path is Git
 fav is Ruby
+deadly skills:
+*Git
+*JS
+*Scala
+*Python
+*Ruby
+*C 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?